StripeFinancialConnections/StripeFinancialConnections/Source/Native/Success/SuccessViewController.swift

```diff
@@ -103,11 +109,40 @@ final class SuccessViewController: UIViewController {
         dataSource
             .analyticsClient
             .logPaneLoaded(pane: .success)
+
+        if showSaveToLinkFailedNotice {
+            dataSource
+                .analyticsClient
+                .log(
+                    eventName: "networking.save_to_link_failed_notice",
+                    pane: .success
+                )
+        }
     }
 }
 
-private func CreateSubtitleText(businessName: String?, isLinkingOneAccount: Bool) -> String {
-    if isLinkingOneAccount {
+private func CreateSubtitleText(
+    businessName: String?,
+    isLinkingOneAccount: Bool,
+    isNetworkingUserFlow: Bool?,
+    saveToLinkWithStripeSucceeded: Bool?
+) -> String {
+    if isNetworkingUserFlow == true && saveToLinkWithStripeSucceeded == true {
+        if let businessName = businessName {
+            return String(
+                format: STPLocalizedString(
+                    "Your account was successfully connected to %@ through Link.",
+                    "The subtitle/description of the success screen that appears when a user is done with the process of connecting their bank account to an application. Now that the bank account is connected, the user will be able to use the bank account for payments. %@ will be replaced by the business name, for example, The Coca-Cola Company."
+                ),
+                businessName
+            )
+        } else {
+            return STPLocalizedString(
+                "Your account was successfully connected to Link.",
+                "The subtitle/description of the success screen that appears when a user is done with the process of connecting their bank account to an application. Now that the bank account is connected, the user will be able to use the bank account for payments."
+            )
+        }
+    } else if isLinkingOneAccount {
```

Comment:
Stripe.js logic: 
- https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blame/6333fce8df72747752fc0dc0b7d3af74fedaff5a/src/linkedAccounts/inner/components/panes/Success/index.tsx#L64-L74

note that the "express" part I confirmed with Shyam that its not necessary